netlify.toml

```diff
@@ -1,14 +1,11 @@
 [build]
-  publish = "public"
-  command = """
-    if [ \"$CONTEXT\" = \"production\" ]; then
-      hugo --baseURL=\"https://istio.io/\"
-    elif [ \"$CONTEXT\" = \"deploy-preview\" ]; then
-      hugo --baseURL=\"$DEPLOY_PRIME_URL/\"
-    else
-      hugo --baseURL=\"https://preliminary.istio.io/\"
-    fi
-  """
+    publish = "public"
+[context.production]
+    command="hugo --baseURL='https://istio.io/latest'" 
+[context.deploy-preview]
+    command="hugo --baseURL='${DEPLOY_PRIME_URL}/'"
+[context.branch-deploy]
+    command="hugo --baseURL='https://preliminary.istio.io/'" 
```

Comment:
feels like you're just spraying things here rather than doing the research and pushing them through thoughtfully fwiw.

* note how preliminary.istio.io has /latest/ URLs too, so if you add that suffix to one, you have to add it to all three.
* the comment before suggested that we shouldn't need to set a baseURL for production but we should set it in the Hugo config file in general.
* I'm not sure the contexts are correct here (and this may require me to find this out in netlify) - preliminary may or may not be a production context on a different site.
The comment